participant/remix.config.js

```diff
@@ -1,7 +1,12 @@
 /**
  * @type {import('@remix-run/dev').AppConfig}
  */
+const { getDependenciesToBundle } = require("@remix-run/dev");
 module.exports = {
   cacheDirectory: "./node_modules/.cache/remix",
   ignoredRouteFiles: ["**/.*", "**/*.css", "**/*.test.{js,jsx,ts,tsx}"],
+  serverDependenciesToBundle: [
+    /strtok3/,
+    ...getDependenciesToBundle("file-type", "token-types", "peek-readable"),
+  ],
```

Comment:
I'm not able to discern why we need this. I read through [the documentation](https://remix.run/docs/en/main/file-conventions/remix-config#serverdependenciestobundle) and I know that we're using `import { fileTypeFromBuffer } from "file-type";` in s3.server.ts, but why do we need to bundle it? Also, why do we need token-types (because it's a dependency of file-types?) and why do we need peek-readable?

Reply:
I don't know, other than the app wouldn't run after i imported `file-type` until i made these changes (all deps of file-type)

Not a huge fan of the package, but the functionality is important to us